api/webhook.go

```diff
@@ -158,8 +158,20 @@ func PostWebhook(c *gin.Context) {
 		return
 	}
 
+	if h.GetEvent() == "repositoryRename" {
```

Comment:
Should this reference the `EventRepositoryRename` constant?

https://github.com/go-vela/types/blob/master/constants/event.go#L24-L25